js/chart/chart-view.js

```diff
@@ -457,7 +457,6 @@ var ChartView = {
   mouseOverMouseOverlay: function(){
     ChartView.properties.mouseOverChart = true;
     if(ChartView.isZoomed()){
-    $('html').css('overflow', 'hidden');
       ChartView.showAllScrollbars();
```

Comment:
Need to remove this line of code so that the overall scrollbar will show when hovering on index chart